common-tooling/README.md

```diff
@@ -0,0 +1,81 @@
+| buildx-cleanup | Cleanup temp files and remove builder at the end of a job |
+| buildx-enabled | Whether to install buildx or not |
+| qemu-image | QEMU static binaries Docker image (e.g. tonistiigi/binfmt:latest) |
+| qemu-platforms | Platforms to install (e.g. arm64,riscv64,arm) | all |
+| qemu-enabled | Whether to install qemu or not |
+| java-version | The Java version to set up | 17 |
+| java-distribution | Java distribution | temurin |
+| java-cache-prefix | Cache key prefix |
+| java-cache-path | Cache path |
+| java-cache-path-add | Additional item for cache path |
+| java-maven-version | The Maven version to set up |
+| java-enabled | Whether to install java or not |
+| python-version | Version range or exact version of Python or PyPy to use, using SemVer's version range syntax. Reads from .python-version if unset. | 3.11 |
+| python-version-file | File containing the Python version to use. Example: .python-version |
+| python-cache | Used to specify a package manager for caching in the default directory. Supported values: pip, pipenv, poetry. |
+| python-architecture | The target architecture (x86, x64) of the Python or PyPy interpreter. |
+| python-check-latest | Set this option if you want the action to check for the latest available version that satisfies the version spec. |
+| python-token | The token used to authenticate when fetching Python distributions from https://github.com/actions/python-versions. When running this action on github.com, the default value is sufficient. When running on GHES, you can pass a personal access token for github.com if you are experiencing rate limiting. |
+| python-cache-dependency-path | Used to specify the path to dependency files. Supports wildcards or a list of file names for caching multiple dependencies. |
+| python-update-environment | Set this option if you want the action to update environment variables. |
+| python-allow-prereleases | When 'true', a version range passed to 'python-version' input will match prerelease versions if no GA versions are found. Only 'x.y' version range is supported for CPython. |
+| python-enabled | Whether to install python or not |
+| overwrite | Defines whether on hosted runners the present version should be overwritten | true |
+| secrets | toJSON passed GitHub secrets |
+| berlin-timezone | Whether to keep the runner at UTC or set Berlin timezone| true|
+
+### Workflow Example
+```yaml
+---
+name: example
+on:
+  pull_request:
+jobs:
+  some-job:
+    runs-on: ubuntu-latest
+    steps:
+    # install any defaults
+    - uses: camunda/infra-global-github-actions/common-tooling@main
+    # install any defaults and configure maven-settings.xml
+    - uses: camunda/infra-global-github-actions/common-tooling@main
+      with:
+        secrets: ${{ toJSON(secrets) }}
+    # only install missing software components
+    - uses: camunda/infra-global-github-actions/common-tooling@main
+      with:
+        overwrite: "false"
+```
```

Comment:
I'd suggest adding more examples here of config options for specific tooling passed (like version of Node etc.), something simple yet giving the users the taste of it.

Reply:
added another example with versions used and comments to their syntax.
Additionally added in the readme the source actions for quick access and usage to their inputs.